helm/g2p-sandbox/values.yaml

```diff
@@ -211,26 +268,51 @@ ph-ee-engine:
       port: 3306
       schema: "tenants"
     ingress:
-      enabled: false           
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx  
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: ops-bk.sandbox.fynarfin.io          
+          paths: 
+          - path: "/"
+            backend:
+              service:
+                name: "ph-ee-operations-app"
+                port:
+                  number: 80
     deployment:
       annotations:
         deployTime: "{{ .Values.deployTime }}"
 
   operations_web:
     enabled: true
     image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-ops-web"
-    imageTag: latest
+    imageTag: "latest"
     hostname: "ops.sandbox.mifos.io"
+    SPRING_PROFILES_ACTIVE: "bb"
     ingress:
-      enabled: false         
+      enabled: true 
+      className: "kong"
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: ops.sandbox.fynarfin.io          
```

Comment:
keep this as mifos.io and change from cookbooks

Reply:
Changed